test/integration/middleware.js

```diff
@@ -683,37 +743,43 @@ module.exports = function (createFn, setup, dismantle) {
     });
   });
 
-  describe("postCreate/Read/Update/Delete - null", () => {
+  describe("postCreate/Read/Update/Delete - undefined", () => {
     let app = createFn();
     let server;
     let customer;
 
-    beforeEach((done) => {
+    before((done) => {
       setup((err) => {
         if (err) {
           return done(err);
         }
 
-        erm.serve(app, db.models.Customer, {
-          postCreate: null,
-          postRead: null,
-          postUpdate: null,
-          postDelete: null,
```

Comment:
⚠️ Passing `null` will now crash, you must now pass either a middleware or middleware array.